python/tvm/meta_schedule/testing/torchbench/run.py

```diff
@@ -124,28 +125,37 @@
 class RunMode(Enum):
     """
     The running mode of this script. Available values are:
-    - tune: Only tune the model and create the tuning database.
+    - extract: Only import the model and extract tuning tasks from it.
+    - tune: Only tune the tasks and create the tuning database.
```

Comment:
Is it possible that we need some other combination of stages, say `extract+tune` or `tune+evaluate`?